client/src/app/kommissions/view/kommission-view.component.html

```diff
@@ -0,0 +1,34 @@
+<div fxLayoutAlign="center">
+  <mat-card fxFlex class="max-width">
+    <mat-card-header>
+      <mat-card-title fxLayout="row">
+        <a mat-button routerLink="/kommissions"><mat-icon>arrow_back</mat-icon></a>
+        <section class="mat-typography">
+          <h1>{{ kommission?.name }}</h1>
+        </section>
+      </mat-card-title>
+    </mat-card-header>
+    <mat-card-content>
+      <p>
+        {{ kommission?.description}}
+      </p>
+      <section class="mat-typography">
+          <h2>Effectif</h2>
+        </section>
+      <p>
+        Les(le/la) barmen(an/aid) actifs de la kommission sont: <span *ngFor="let barman of getBarmen(true)">{{ barman.nickname }}</span>
```

Comment:
Can we just use 'Les barmen actifs ...' instead of handling every cases? It's just a proposition to simplify readibility :smiley: